### PR TITLE
feature: recover folder cloneable option in folder create/setting modal.

### PR DESCRIPTION
**Pull request by lizable**
> [!IMPORTANT]  
> Please merge this PR after https://github.com/lablup/backend.ai/pull/1749 is merged.

This PR resolves showing up new type of project (`MODEL_STORE`) in vfolder creation so that user could create model type vfolder on corresponding project.

**Checklist:** (if applicable)

- [ ] Mention to the original issue
- [ ] Documentation
- [x] Minimum required manager version
- [ ] Specific setting for review (eg., KB link, endpoint or how to setup)
- [ ] Minimum requirements to check during review
- [x] Test case(s) to demonstrate the difference of before/after

     ```
     $ ./backend.ai vfolder create -g model-store --usage-mode model testmodel local:volume1
     ```
     (This command will create project "model-store" and model type vfolder names "testmodel" )

※ NOTE: Only admin user can create group vfolder

- [ ] Check whether you can create "General" usage mode vfolder with both user/group type.
- [ ] Check whether you can create "MODEL_STORE" usage mode vfolder with both user/group type.
- [ ] Check "MODEL_STORE" usage mode vfolder is visible in models tab. (admin only)
- [ ] Check only admin can select "MODEL_STORE" project by the project selector of the header.

### CODE REVIEW

File: src/lib/backend.ai-client-esm.ts [hunk 1]
```diff
@@ -4060,6 +4061,7 @@ class Group {
    * };
    */
   async list(
+    type = ["GENERAL", "MODEL_STORE"],
```
Comment (yomybaby) [inline src/lib/backend.ai-client-esm.ts]: For better compatibility, it's better to add a new argument as the last argument.

**Review by yomybaby — CHANGES_REQUESTED** [review]

**Review by agatha197 — APPROVED** [review]
LGTM

**Review by yomybaby — APPROVED** [review]
LGTM

**Review by agatha197 — APPROVED** [review]
LGTM

**Pull request merged**